pydrive2/test/test_util.py

```diff
@@ -11,7 +11,7 @@
 newline_pattern = re.compile(r"[\r\n]")
 
 GDRIVE_USER_CREDENTIALS_DATA = "GDRIVE_USER_CREDENTIALS_DATA"
-DEFAULT_USER_CREDENTIALS_FILE = "credentials/default.dat"
+DEFAULT_USER_CREDENTIALS_FILE = "credentials/credentials.json"
```

Comment:
CREDENTIALS in this case means saved credential not p12/service json file I think